webhook/pom.xml

```diff
@@ -47,6 +47,16 @@
 			<groupId>org.liquibase</groupId>
 			<artifactId>liquibase-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.h2database</groupId>
```

Comment:
I think this dependency is unused.

Reply:
I use this dependency for integration test

Reply:
For integration test, I think you are using PostgreSQL DB not H2 DB. Is it correct?

`@Bean(destroyMethod = "stop")
    public PostgreSQLContainer<?> postgresContainer() {
        return new PostgreSQLContainer<>("postgres:16")
            .withReuse(true);
    }`

Reply:
Fixed